.mocharc.yml

```diff
@@ -0,0 +1,4 @@
+exclude:
+  - test/helpers/scripts/*.js
```

Comment:
@fearphage this works and I don't have any blockers for it. Although this implementation will require periodic additions to exclusions for any other non test files that may be added down the road; whereas previously nothing should have required updates as  it leveraged a file glob for determining what is/isn't a test. 

If this is what your preference is though fine by me and we can merge this in.

Reply:
This is better than the alternative that you caught -- a fie was lying dormant in the test directory due to being poorly named.

Reply:
True, even if that's severely unlikely in my opinion. Either way this is a departure from our current implementation which I was trying to avoid at all costs where possible.

Reply:
This is the defacto standard for most projects. `mocha` is generally all you need to run tests. Things should work in ways people expect them to.